assets/js/modules/idea-hub/datastore/draft-ideas.js

```diff
@@ -34,12 +34,6 @@ const fetchCreateIdeaDraftPostStore = createFetchStore( {
 	controlCallback: ( { idea } ) => {
 		return API.set( 'modules', 'idea-hub', 'create-idea-draft-post', { idea } );
 	},
-	reducerCallback: ( state, ideaDraftPost ) => {
-		return {
-			...state,
-			ideaDraftPost,
-		};
-	},
```

Comment:
@ivankruchkoff it turns out that we still need this reducerCallback function, but instead of adding the newly created draft to its own state property, we need to append it to the `draftPostIdeas` array:

```javascript
reducerCallback: ( state, ideaDraftPost ) => {
    return {
        ...state,
        draftPostIdeas: [ ...( state.draftPostIdeas || [] ), ideaDraftPost ],
    };
},
```

Could you please add it?